source/common/util/counter.ts

```diff
@@ -83,10 +94,8 @@ export function countWords (ast: ASTNode, from = 0, to?: number): number {
  *
  * @return  {number}         The number of characters in the file
  */
-export function countChars (ast: ASTNode, from = 0, to?: number): number {
-  return getCleanedWords(ast, from, to)
-    .map(w => w.length)
-    .reduce((prev, cur) => prev + cur, 0)
+export function countChars (ast: ASTNode, locale: string | undefined, from = 0, to?: number): number {
```

Comment:
```suggestion
export function countChars (ast: ASTNode, locale?: string, from = 0, to?: number): number {
```

Also, same question as above